### PR TITLE
KV example app, and a test for prefixes

### PR DESCRIPTION
**Pull request by levb**
#### Summary
Added a test and an example app, can not reproduce the issue in the ticket.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46108

### CODE REVIEW

File: server/builtin/debug.go [hunk 1]
```diff
@@ -106,6 +106,21 @@ func (a *builtinApp) debugBase64KeyField(loc *i18n.Localizer) apps.Field {
 			ID:    "field.kv.base64key.hint",
 			Other: "[ base64-encoded key ]",
 		}),
+	}
+}
+
+func (a *builtinApp) debugBase64Field(loc *i18n.Localizer) apps.Field {
```
Comment (levb) [inline server/builtin/debug.go:112]: added this back to `/apps debug kv list` - not sure why it was dropped.

**Review by catalintomai — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -39,79 +39,35 @@ commands:
       - aws-s3/copy:
           from: << parameters.filename >>
           to: << parameters.bucket >>
-          arguments: '--acl public-read --cache-control no-cache'
-
-  install-golangci-lint:
```
Comment (hanzei) [inline .circleci/config.yml:44]: Why is this not longer needed?

Comment (levb) [reply .circleci/config.yml]: Sorry, this is from another PR, all rebased now.

File: examples/goapp/kv/app.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+// static is preloaded with the contents of the ./static directory.
+//go:embed static
+var static embed.FS
+
+// main starts the app, as a standalone HTTP server. Use $PORT and $ROOT_URL to
+// customize.
+func main() {
+	// Create the app, add `send` to the app's command and the channel header.
+	goapp.MakeAppOrPanic(
```
Comment (hanzei) [inline examples/goapp/kv/app.go:18]: Unrelated to this PR: The naming pattern usually used in Go for functions that panic on error is `MustX`. Should we adopt that for `goapp`?

File: examples/goapp/kv/app.go [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+)
+
+var get = goapp.MakeBindableFormOrPanic("get",
+	apps.Form{
+		Title: "Get a value from the KV store",
+		Icon:  "icon.png",
+		Fields: []apps.Field{
+			{
+				Name:          "prefix",
+				Description:   "The namespace prefix to use, just 2 charachters, don't even ask why...",
```
Comment (hanzei) [inline examples/goapp/kv/app.go:92]: 😅

**Review by hanzei — APPROVED** [review]
Nice work, thanks Lev:+1:

**Pull request merged**